api/v1beta1/client.go

```diff
@@ -87,6 +107,37 @@ func getItems(list client.ObjectList) []client.Object {
 	return items
 }
 
+// OVNControllerNamespaceMapFunc - OVNController Watch Function
+func OVNControllerNamespaceMapFunc(crs client.ObjectList, reader client.Reader) handler.MapFunc {
```

Comment:
why do you have to copy it instead of reusing the existing one? (Maybe rename it.)

Reply:
Yeah, will try to rename it so OVNCluster and OVNDBCluster uses same genericFunction